### ISSUE TITLE
Write documentation

### ISSUE THREAD

**Issue by kyrias** [hyperboria/python-cjdns#5]
Document all functions using Sphinx and host the documentation on RTD

**Comment by thefinn93** [comment]
If we're talking about all the functions of this library, sure, but I feel like the cjdns API docs should go elsewhere. Maybe we can format them in a nice pythonic way and put em up, but the content should be language-agnostic.

**Comment by kyrias** [comment]
Yeah, I meant this library, though writing Sphinx docs for the cjdns admin API would be great as well
